### PR TITLE
haskellPackages.zeromq4-haskell_0_8_0: unbreak

### PR DESCRIPTION
**Pull request by vaibhavsagar**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by vaibhavsagar** [comment]
Tested with `nix-build -A haskellPackages.zeromq4-haskell_0_8_0`.

**Comment by xeji** [comment]
@GrahamcOfBorg build haskellPackages.zeromq4-haskell_0_8_0

File: pkgs/development/haskell-modules/hackage-packages.nix [hunk 1]
```diff
@@ -250373,7 +250373,6 @@ self: {
        description = "Bindings to ZeroMQ 4.x";
        license = stdenv.lib.licenses.mit;
        hydraPlatforms = stdenv.lib.platforms.none;
-       broken = true;
```
Comment (peti) [inline pkgs/development/haskell-modules/hackage-packages.nix]: Do not edit this file! Note the comment at the very top.

**Review by peti — CHANGES_REQUESTED** [review]
:-1:

File: pkgs/development/haskell-modules/hackage-packages.nix (hunk 1 above)
Comment (vaibhavsagar) [reply pkgs/development/haskell-modules/hackage-packages.nix]: Okay, what file should I be editing instead? The comment at the top of the file doesn't have this information.

Comment (vaibhavsagar) [reply pkgs/development/haskell-modules/hackage-packages.nix]: My first guess would be `configuration-hackage2nix.yaml`, but there doesn't seem to be an entry for  `zeromq4-haskell_0_8_0`.

**Pull request closed without merging**